src/AutoConstructor.Generator/Core/IsExternalInit.cs

```diff
@@ -0,0 +1,17 @@
+// <auto-generated />
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices;
+
+/// <summary>
+/// Reserved to be used by the compiler for tracking metadata.
+/// This class should not be used by developers in source code.
+/// </summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
+internal static class IsExternalInit
```

Comment:
You can avoid manually copying types like this and just reference [PolySharp](https://github.com/Sergio0694/PolySharp) instead. That will also include all other polyfills for language features, so you can enable C# 11 (or C# 12, or whatever) and also happily use all other features that don't require runtime support (eg. patterns, list patterns, collection expressions, required properties, etc.) 🙂

That package is a generator only, so you can use it just fine from your own generator (no .dll-s to copy).